test/dialogmote/MotehistorikkTest.tsx

```diff
@@ -151,10 +150,12 @@ describe("Historiske dialogmøter", () => {
 
     const buttons = screen.getAllByRole("button");
     expect(buttons).to.have.length(2);
-    expect(buttons[0].textContent).to.equal(
+    expect(buttons[0].textContent).to.contain(
```

Comment:
Må bruke `contain` her, fordi chevronen har teksten "vis mer" så for maskinen blir teksten på `accordion`en "Vis merReferat fra møte..."